PIM/UserInterface/02a_List.md

```diff
@@ -0,0 +1,1419 @@
+
+[comment]: <> (not working -> if I want to edit an offer, a window is displayed loading all the time...)
+
+The list displays all created offers to this product. All fields are read-only.
+
+- *SKU*   
+  Stock Keeping Unit. Identification number of the product. In the *Create product* view, **TO BE GENERATED** is displayed.
+
+- *Status*   
+  Connection status of the offer. In the *Create product* view, **Offline** is displayed.
+
+- *Pending status/Errors*   
+  Indication of the pending status or an error for this offer. The selected status (**Active**/**Inactive**/**Offline**) is displayed if you have changed the offer status via the editing toolbar.
+
+  [comment]: <> (when is an error displayed here? what is displayed? The Error code or message?)
+
+- *Connection*   
+  Selected connection for the offer.
+
+- *Attribute Set*   
+  Selected attribute set for the offer.
+
+- *Last edited on*   
+  Date and time of the last modification. In the *Create product* view, no date is displayed.
+
+- *Modified by*   
+  Name and username of the user who modified the product. In the *Create product* view, no name is displayed.
+
+- *Change tracking mode*   
+  Change tracking mode (ETL mode) of the offer. The following modes can be displayed:
+  - **manual**
+  - **semi-automatic**
+  - **semi-automatic, changes must be confirmed by another user**   
+  - **automatic**
+
+
+- *ID*   
+  Product identification number. The ID number is automatically assigned by the system.
+
+- *Created on*   
+  Date and time of the creation. In the *Create product* view, no date is displayed.
+
+- *Created by*   
+  Name and username of the user who created the product. In the *Create product* view, no name is displayed.
+
+- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)   
+  Click this button to create a new offer. The [Add to a single connection ![Add to a single connection](../../Assets/Icons/Document.png "[Single connection]") ] button and the [Add to multiple connections ![Multiple connections](../../Assets/Icons/Documents.png "[Multiple connections]") ] button are displayed.
+
+
+### Add an offer in PIM
+*PIM > Products > Tab LIST > Button Add > Button CREATE > Tab Offers > Button Add*
+
+![Add](../../Assets/Screenshots/PIM/Products/List/Offers/AddCreate.png "[Add]")
+
+- [Add to a single connection ![Document](../../Assets/Icons/Document.png "[Document]") ]    
+  Click this button to add a single variant to the product. The *Add to a single connection* window is displayed.
+
+- [Add to multiple connections ![Documents](../../Assets/Icons/Documents.png "[Documents]") ]    
+  Click this button to add multiple variants to the product. The *Add to multiple connections* wizard window is displayed.
+
+- ![Cancel](../../Assets/Icons/Cross01.png "[Cancel]") (Cancel)   
+  Click this button to cancel the connection creation and close the current view.
+
+
+### Add to a single connection
```

Comment:
```suggestion
### Add to single connection
```
Button heißt eigentlich Add to single connection (oder stimmt es nicht ganz?) -> Evtl. Links korrigieren